src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <div class="header__logo-container">
+          <a class="header__link" href="#home">
+            <img src="images/logo.png" alt="Moyo-logo"
+            class="header__logo">
+          </a>
+        </div>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__list_item">
+              <a class="nav__link is-active" href="#">Apple</a>
+            </li>
+            <li class="nav__list_item">
+              <a class="nav__link"  href="#">Samsung</a>
+            </li>
+            <li class="nav__list_item">
+              <a class="nav__link"  href="#">Smartphones</a>
+            </li>
+            <li class="nav__list_item">
+              <a
+              data-qa="hover"
+              class="nav__link"
+              href="#"
+              >Laptops & Computers</a>
```

Comment:
Use formatting according to styleguide in the comments above. Move `Laptops & Computers` on the new line and add spaces for the attributes